src/rules/javascript/banned_identifiers.js

```diff
@@ -0,0 +1,17 @@
+import { BANNED_IDENTIFIERS } from 'const';
+
+export default function(context) {
+  return {
+    Identifier: function(node) {
```

Comment:
I notice my checks are slightly different to say the mozindexeddb prop ones  - we should maybe see if we can DRY these type of checks up - assuming it does make sense.